src/nnet3/nnet-normalize-component.cc

```diff
@@ -606,6 +757,22 @@ void BatchNormComponent::Read(std::istream &is, bool binary) {
   stats_sumsq_.AddVecVec(1.0, stats_sum_, stats_sum_, 1.0);
   stats_sum_.Scale(count_);
   stats_sumsq_.Scale(count_);
+  ExpectToken(is, binary, "<BatchRenorm>");
```

Comment:
this would have broken any existing BatchNormComponents on disk.

Reply:
yes, back-level compatibility is needed here.